### PR TITLE
adding s3 backup capability need to test to make sure we have everything

### PR DESCRIPTION
**Pull request by simplyadrian**
in place.

### CODE REVIEW

File: restore.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+if [[ -n "$PRODUCT" ]] && [[ ! -e $JENKINS_HOME/config.xml ]]
+then
+  aws --region us-west-2 s3 cp "s3://<some_bucket_name>/$PRODUCT/jenkins/jenkins_data.tgz" /tmp/restored.tgz &&\
```
Comment (meganlkm) [inline restore.sh]: hard coded region?

Comment (simplyadrian) [reply restore.sh]: fixed

File: fixup_backup.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+if [[ -n "$PRODUCT" ]]
+then
```
Comment (meganlkm) [inline fixup_backup.sh]: this is weird. why not `if [[ ! -n "$PRODUCT" ]]` ?

Comment (simplyadrian) [reply fixup_backup.sh]: oops

Comment (simplyadrian) [reply fixup_backup.sh]: we are checking if $PRODUCT is set and if it is perform the action. See my comment above around this. If we don't want to support this or want to have to pass this variable we can remove it all together.

**Comment by simplyadrian** [comment]
@sapiensantiquus and @meganlkm do we want to merge this ? Has this passed the deployment test @meganlkm ?

**Pull request merged**